lib/seeds/course_seeder.rb

```diff
@@ -5,11 +5,13 @@ module Seeds
   class CourseSeeder
     include Seeds::Helpers
 
-    attr_accessor :identifier_uuid, :title, :description, :position
+    attr_accessor :identifier_uuid, :title, :description, :position, :show_on_homepage, :badge_uri
 
-    def initialize(path, position)
+    def initialize(path, position, badge_uri: nil, show_on_homepage: false)
       @path = path
       @position = position
+      @badge_uri = badge_uri
```

Comment:
We can omit the `badge_uri` and `show_on_home_page` instance variables. The block initialization will take care of setting these attributes for us. 

We'll just have to set the default for `show_on_home_page` in the course method below for the not null constraint:
`course.show_on_homepage = show_on_homepage || false`